scalability_and_performance/sno-du-enabling-workload-partitioning-on-single-node-openshift.adoc

```diff
@@ -0,0 +1,51 @@
+
+When you use workload partitioning, the CPU resources used by {product-title} for cluster management are isolated to a partitioned set of CPU resources on a single node cluster with a DU profile applied. This falls broadly into two categories:
+
+* Isolates cluster management functions to the defined number of CPUs. All cluster management functions operate solely on that `cpuset`.
+
+* Tunes the cluster configuration (with the applied DU profile) so the actual CPU usage fits within the assigned `cpuset`.
+
+The minimum number of reserved CPUs required for the management partition for a single node cluster is four CPU Hyper threads (HTs). Inclusion of Operators or workloads outside of the set of accepted management pods requires
+additional CPU HTs.
+
+Workload partitioning isolates the workloads away from the non-management workloads using the normal scheduling capabilities of Kubernetes to manage the number of pods that can be placed onto those cores, and avoids mixing cluster management workloads and user workloads.
+
+When you use workload partitioning, the CPU resources used by {product-title} for cluster management are isolated to a partitioned set of CPU resources on a single node cluster with a DU profile applied. This falls broadly into two categories:
+
+* Isolates cluster management functions to the defined number of CPUs. All cluster management functions operate solely on that `cpuset`.
+
+* Tunes the cluster configuration (with the applied DU profile) so the actual CPU usage fits within the assigned `cpuset`.
+
+The minimum number of reserved CPUs required for the management partition for a single node cluster is four CPU HTs. Inclusion of Operators or workloads outside of the set of accepted management pods requires additional CPU HTs.
+
+Workload partitioning isolates the workloads away from the non-management workloads using the normal scheduling capabilities of Kubernetes to manage the number of pods that can be placed onto those cores, and avoids mixing cluster management workloads and user workloads.
+
+To fully leverage workload partitioning, you need to install the Performance Addon Operator and apply the performance profile.
+
+The concept of cluster management workloads is flexible and can encompass:
+
+* All {product-title} core components necessary to run the cluster.
+
+* Any add-on Operators necessary to make up the platform as defined by the customer.
+
+* Operators or other components from third-party vendors that the customer deems as management rather than operational.
+
+** Adding management workloads might require additional CPU resources to be added to the partition.
+
+Workload partitioning introduces a new extended resource of `<workload-type>.workload.openshift.io/cores` for each CPU pool, workload-type, defined in the configuration file. Kubelet advertises these new resources. When workload partitioning is enabled, it represents the CPU capacity of the host, not just the CPU pool.
```

Comment:
```suggestion
Workload partitioning introduces a new extended resource of `<workload-type>.workload.openshift.io/cores` for each defined CPU pool (workload-type). Kubelet advertises these new resources and CPU requests by pods allocated to the pool will be accounted for within the corresponding resource rather than the typical `cpu` resource. When workload partitioning is enabled the `<workload-type>.workload.openshift.io/cores` resource allows access to the CPU capacity of the host, not just the default CPU pool.
```